src/test/ui/async-await/dont-suggest-missing-await.stderr

```diff
@@ -1,6 +1,9 @@
 error[E0308]: mismatched types
   --> $DIR/dont-suggest-missing-await.rs:14:18
    |
+LL | async fn make_u32() -> u32 {
```

Comment:
As a user: the order here is confusing to me.
Maybe this will read better as a note after the expected/found?
Like:
```
note: the found opaque type is the `Output` of this async fn
```